bezpy/mt/site.py

```diff
@@ -243,8 +243,19 @@ def download_waveforms(self):
 
         # Magnetic Field
         self.waveforms[["FE", "FN", "FZ"]] *= 0.01  # nT
-        # Electric Field
-        self.waveforms[["QN", "QE"]] *= 2.44141221047903e-05  # mV/km
+        # Electric voltage
+        self.waveforms[["QN", "QE"]] *= 2.44141221047903e-06  # mV
+        # Electric Field (set default length of dipole as 100m)
+        self.waveforms[["QN", "QE"]] *= (1000.0/100.0)  # mV/km
+        for runid in self.runlist:
+            try:
+                mask = ((self.waveforms.index>self.runinfo[runid]['Start']) & (self.waveforms.index<self.runinfo[runid]['End']))
```

Comment:
Put spaces around the '>' symbols so it is easier to see the comparison. Also need to break it up into multiple lines.

Reply:
Thanks